src/autobot/autobotmodule.cpp

```diff
@@ -69,6 +75,11 @@ void AutobotModule::resolveImports()
     if (ar) {
         ar->reg(std::make_shared<AutobotActions>());
     }
+
+    auto api = modularity::ioc()->resolve<IApiRegister>(moduleName());
+    if (ar) {
```

Comment:
Oops, this should have been `if (api) {`, shouldn't it?

Reply:
> Oops, this should have been `if (api) {`, shouldn't it?

Oops...
I will fix it in a next PR